lib/engine/game/base.rb

```diff
@@ -943,6 +945,7 @@ def next_round!
           when Round::Stock
             @operating_rounds = @phase.operating_rounds
             reorder_players
+            sr_finished
```

Comment:
This doesn't seem to be used, otherwise it's fine.

Reply:
Ah, solution evolved over time....

Have removed that method in the new version. Had to rebase it so it was a push force.